test/test_utils/sagemaker.py

```diff
@@ -62,7 +62,8 @@ def assign_sagemaker_remote_job_instance_type(image):
     elif "inference-neuron" in image:
         return "ml.inf1.xlarge"
     elif "gpu" in image:
-        return "ml.p3.8xlarge"
+        ## TODO: Revert
```

Comment:
Will revert

Reply:
not a bad idea to make the default instance type g5 for pt/tf inference

Reply:
Made the change, thanks!